docs/deployment.md

```diff
@@ -86,7 +86,7 @@ Options:
   --ssl-certfile TEXT             SSL certificate file
   --ssl-keyfile-password TEXT     SSL keyfile password
   --ssl-version INTEGER           SSL version to use (see stdlib ssl module's)
-                                  [default: 2]
+                                  [default: 17]
```

Comment:
this corresponds to ssl.PROTOCOL_TLS_SERVER right ?

Reply:
yeah I'm not sure what to do about this flag

Reply:
I think this CLI flag should be changed to take either 1.2 or 1.3 and use a similar trick to https://github.com/encode/httpx/blob/master/httpx/_compat.py#L16

Reply:
> I think this CLI flag should be changed to take either "1.2" or "1.3" and use a similar trick to https://github.com/encode/httpx/blob/master/httpx/_compat.py#L16

makes sense, but maybe let's keep that for a seperate PR, I wonder what gunicorn does about it, I remember I read about lenghty debate there but nothing was very actionnable iirc